FirebaseVertexAI/Sources/GenerativeModel.swift

```diff
@@ -306,7 +306,7 @@ public final class GenerativeModel {
   }
 }
 
-/// See ``GenerativeModel/countTokens(_:)-9spwl``.
+/// See ``GenerativeModel/countTokens(_:)``.
```

Comment:
How about this
```
/// An error thrown in `countTokens(_:)`.
```
since the linking won't work without the hash.

Reply:
Done.